### ISSUE TITLE
Proposed RFC Suggestion: SIG Information

### ISSUE THREAD

**Issue by JustFinchy** [o3de/community#152]
### Summary:

There have been reports that members have a hard time finding how they can contribute code, how to assist sigs, and find information about sigs themselves. We had reached out to the community and one of our community members Nodnam said “I have no idea what a SIG is or if I can join it. I’m only on this discord server because I’m interested in engine dev and I don’t really have any outside knowledge of the engine besides this discord.” and another member Andre said “I don’t understand the SIGs at all or how to contribute to them. I’ve joined in one of the meetings, but I felt like an outsider looking in and was fairly lost.”

This proposal is to create a document that lives on the GitHub, Discord (In a channel and on each SIG) along with on the website that describes what a SIG is. This document will list each of the SIGs and their chair/co-chairs along with related links. It will also list what they deal with and their area of scope. Finally, it will list how to contribute to the community with examples. 

### What is the motivation for this suggestion?
Currently the SIG's follow different outlines and some list their repo's, while others do not list anything. New members often find themselves very lost. 

### Suggestion design description:
This proposal is to create a document that lives on the GitHub, along with on the website that describes what a SIG is. This document will list each of the SIGs and their chair/co-chairs along with related links. It will also list what they deal with and their area of scope. Finally, it will list how to contribute to the community with examples. 

An example can be found here
https://github.com/o3de/sig-ui-ux#readme

**Comment by lmbr-pip** [comment]
Am slightly confused about what is being proposed.

Some SIGs have really good READMEs ie https://github.com/o3de/sig-ui-ux/blob/main/README.md, https://github.com/o3de/sig-simulation/README.md or https://github.com/o3de/sig-network/README.md

I would counter propose:
* That we have a comment template for SIG readme based on the best things from the current READMEs.
* We get Chair(s) to update SIG README to this new common template, so it provides what the community needs, and make it something the SIGs own moving forward.
* Find mechanisms to highlight SIG READMEs (ie through discord links etc) so everyone can find whats needed.

Note: I've also proposed some README style guides to community here: https://github.com/o3de/community/pull/151 to try and bring some structure to that repro, which is the best "common" place for process docs we have.

I think ideally we want something more organized like https://wiki.blender.org/wiki/Process but we need to build on what we have.

**Comment by aFinchy** [comment]
@lmbr-pip Yes, Its more or less a Common template suggestion in all the repos as seems a lot of repos do not have any information or have very little information. 


Though fully agreed on everything you mentioned above.
